examples/tutorial/x/blog/handlers_test.go

```diff
@@ -208,6 +209,45 @@ func testHandlerDeliver(t *testing.T, testcases []testcase) {
 		}
 	}
 }
+
```

Comment:
this seems correct and useful for handlers where we test the outcome of sequences of actions, but I think we don't need this for queries.

Just like the TestQuery function below, expand a bit and add three cases for the three data types.